recipes/lazy_property/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "lazy_property" %}
```

Comment:
Can this be `lazy-property` so the name is the same as on pypi?

Reply:
Done.